attestation-service/verifier/src/cca/mod.rs

```diff
@@ -57,10 +57,17 @@ fn my_evidence_builder(
 impl Verifier for CCA {
     async fn evaluate(
         &self,
-        nonce: String,
-        attestation: &Attestation,
+        evidence: &[u8],
+        report_data: Option<&[u8]>,
+        init_data_hash: Option<&[u8]>,
```

Comment:
Hi @chendave  Hi Dave, I've changed the API of verifier of `CCA`. PTL if it looks good to you.

BTW, it will now change the behavior of current CoCoAS.